src/binding/c/lottieanimation_capi.cpp

```diff
@@ -148,4 +150,71 @@ lottie_animation_render_flush(Lottie_Animation_S *animation)
     return animation->mBufferRef;
 }
 
+LOT_EXPORT void
+lottie_animation_property_override(Lottie_Animation_S *animation,
+                                   const char *keypath,
+                                   const char *type,
+                                   ...)
+{
+    va_list prop;
+    va_start(prop, type);
+
+    rlottie::Property prop_type = rlottie::Property::FillColor;  // Temporary initialize.
+    if (!strcmp(type, "FillColor"))           prop_type = rlottie::Property::FillColor;
```

Comment:
why give property as a string rather than enum ??

Reply:
There is no particular reason. Using an enum is clearer and better to look at.
I will make new patch soon.